meson.build

```diff
@@ -150,6 +150,23 @@ if get_option('buildtype').startswith('debug')
             '-D_LIBCPP_DEBUG=1',
         ]
     endif
+
+    # Enable Loguru stack traces if supported
+    stacktrace_headers = ['cxxabi.h', 'dlfcn.h', 'execinfo.h']
+
+    all_stacktrace_headers_found = true
+
+    foreach header : stacktrace_headers
+        if not cxx.has_header(header)
+            message('@0@ not found'.format(header))
+            all_stacktrace_headers_found = false
+        endif
+    endforeach
+
+    if all_stacktrace_headers_found
+        message('Enabling Loguru stack traces')
+        add_project_arguments('-DLOGURU_STACKTRACES=1', language: 'cpp')
+    endif
```

Comment:
Suggest Meson's conf header define approach (see `src/config.h`)

```c++
#mesondefine LOGURU_STACKTRACES
```

Then here in the meson.build:

```python
conf_data.set10('LOGURU_STACKTRACES', all_stacktrace_headers_found)
```

The `message` can summarized at the end of the `meson setup` phase. something like:

```python
summary('Loguru stack traces', all_stacktrace_headers_found)
```

Reply:
that didn't work, since loguru.cpp doesn't include config.h, so I put it in the build rather than hack up the loguru base.